packages/outline-docs/package.json

```diff
@@ -23,7 +23,7 @@
   },
   "license": "BSD-3-Clause",
   "scripts": {
-    "build": "node ../../scripts/build.js",
+    "build": "node ../../scripts/build.js && cp -r src dist",
```

Comment:
We should keep `package.json` simple. Any additional actions required for `build` should be inside `build.js` script

Reply:
Since I don't think we want this particular step to be done for all packages, do you have any recommendations on technique to allow per package additions/augmentations to the build script?

For example, we could have a convention where a `scripts/build.js` file is looked for in each package and if found executed after all the commands in the common `build.js` script.